topbar.css

```diff
@@ -37,4 +37,7 @@ body {
 
 #topbar li a:hover {
     background-color: #777777;
+}
+.w-40{
+    width: 40% !important;
```

Comment:
What does this do?

Reply:
for custom button width, I used this one